### ISSUE TITLE
Monk Character Sheet Issues

### ISSUE THREAD

**Issue by Sarsem** [gizmomathboy/hyperborea_apps#8]
New issues, with Monk: name has hyphen in it, isn't properly spelled out; empty hand/caestuses attack rate and damage aren't correct; FA is 6, should be 8; class abilities include "accurate strike" (from 2e), but not "extraordinary" strength or dexterity feats, enlisted henchmen; empty hand is described as per 2e
[Sit'u Gooch - Monk 9.pdf](https://github.com/gizmomathboy/hyperborea_apps/files/9229818/Sit.u.Gooch.-.Monk.9.pdf)

**Comment by gizmomathboy** [comment]
I think everything is correct except name. That's a problem with my pdf creating tool (pdftk).

Also, MV is wrong on sheet. Base MV is 50 with short bursts of 60.

**Comment by Sarsem** [comment]
Looks like the Empty Hand attack rate is still 2e, not 3e (1/1 vs. 2/1), and damage is missing the 3d4.

**Comment by gizmomathboy** [comment]
Can you run it again? I think I have everything sorted out but the name and MV.
[monk-level-9.pdf](https://github.com/gizmomathboy/hyperborea_apps/files/9252261/monk-level-9.pdf)

**Comment by Sarsem** [comment]
It looks closer: you're right about MV, but also missing are Enlist Henchmen and Extraordinary Strength, and the Empty Hand is properly listed under Class Abilities, but not reflected in the Weapon Combat table (caestuses just show attack rate 1/1, +1 damage, when it should be 2/1, 3d4+1).

**Comment by gizmomathboy** [comment]
Ok, sorted out Enlisted Henchmen.

I don't see Extraordinary ST for monk, just DX in the 3e pdf.

Yeah, I need to sort out the MV stuff, and the empty hand/caestus stuff for melee weapons. That's a bit more involved :-)

**Comment by Sarsem** [comment]
Hmm, p.80 of my 3e PDF states:
"EXTRAORDINARY
+8% chance to perform extraordinary feats of strength and dexterity (see Chapter 3: Statistics, strength and dexterity)."

**Comment by gizmomathboy** [comment]
There is an update to the players manual (2022-02-02) that removed the ST part of that.

I think I'll need to check the printed version to see what it has.

**Comment by Sarsem** [comment]
Whoops, totally missed the update: just downloaded it, and confirmed the removal of strength from Extraordinary. Thanks!

**Comment by gizmomathboy** [comment]
No worries. I think I just happened to be in drivethrurpg and just happened to look in my library for updates.

I have the MV stuff "working" locally. I have it in the box as 50 [60].

I tried "50 (burst 60)" but it looked to small.

Still wrangling with empty hand stuff. Getting closer.

**Comment by Sarsem** [comment]
Yeah, I haven't received my hardcopies yet, as I went in for a copy of the leatherette versions. Happy to keep testing, appreciate your work!

**Comment by gizmomathboy** [comment]
I did confirm that the printed version matches the latest pdf.

Also, I think I sorted out the caestuses and empty hand stuff for the weapon tables.

I will be opening another issue for the Enlist Henchmen and other 3e changes for class abilities.

Give things a try and let me know if they are working as expected...for a 9th level monk. I can't recall where I left off with the class ability updates.
